hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/HddsUtils.java

```diff
@@ -612,4 +615,11 @@ public static String format(List<String> nodes) {
   public static long getShutDownTimeOut(ConfigurationSource conf) {
     return conf.getObject(OzoneServiceConfig.class).getServiceShutdownTimeout();
   }
+
+  /**
+   * Utility method to round up bytes into the nearest MB.
+   */
+  public static int roundupMb(long bytes) {
+    return Math.toIntExact((bytes - 1) / ONE_MB + 1);
```

Comment:
This will always compute to a long, and this will not be a floating point number. Also I think there should be a utility to perform this.